website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -86,6 +86,7 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
 | `data.attributes.vcs-repo.branch`             | string  | (nothing) | The repository branch that Terraform will execute from. If omitted or submitted as an empty string, this defaults to the repository's default branch (e.g. `master`) .                                                                                                                                                                                                                                                                                                                                          |
 | `data.attributes.vcs-repo.ingress-submodules` | boolean | `false`   | Whether submodules should be fetched when cloning the VCS repository.                                                                                                                                                                                                                                                                                                                                                                                                                                           |
 | `data.attributes.vcs-repo.identifier`         | string  |           | A reference to your VCS repository in the format :org/:repo where :org and :repo refer to the organization and repository in your VCS provider. The format for Azure DevOps is `:org/:project/_git/:repo`.                                                                                                                                                                                                                                                                                                      |
+| `data.attributes.vcs-repo.tags-regex`         | string  | (nothing) | A regular expression used to match Git tags. When this value is present and a VCS event is received that contains a matching Git tag for the regular expression, a Terraform Cloud run will be triggered.                                                                                                                                                                                                                                                                                                       |
```

Comment:
Hi @hashimoon 

I'm Matthew, new technical writer on the Terraform team. Overall, very good stuff here. I just have a couple of pretty small wording changes.

For the Description of `data.attributes.vcs-repo.tags-regex`, I'd suggest rewording to "Terraform Cloud triggers a run when this value is present and a VCS event occurs that contains a matching Git tag for the regular expression."

For the Description of `data.attributes.working-directory`, I'd suggest rewording to "A relative path from which Terraform executes. The path defaults to your repository's root and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository."

Thanks very much for your time and effort.

Reply:
One tweak to the `data.attributes.working-directory` section. Maybe: 
The path defaults to your repository's root. Set it to a subdirectory matching the environment when multiple environments exist within the same repository."

Reply:
Hi @mgarrell777 and @judithpatudith it's always great to get input on how to improve documentation. However working directory is not in the scope of changes this PR targets. Even thou this is a small change and it would hurt the readability of the PR, having changes unrelated to the goal of the PR makes it harder to distinguish what really matters for - in this case `tags-regex` field. Also if we'd have to revert changes we would have to manually scan the changes in order to identify the ones that are related to the `tags-regex` field and not to remove all the improvements.

WDYT

Reply:
@mgarrell777 thanks for the suggestion for `tags-regex` -> applied here 0fd41d584b222d61cfc06506b7f1a00ecf8c4e13